packages/react-components/src/molecules/TabbedCard.tsx

```diff
@@ -51,7 +55,7 @@ const TabbedCard = <T extends object>({
     getShowMoreText && truncateFrom && items.length > truncateFrom;
 
   return (
-    <Card padding={false}>
+    <Card padding={false} overrideStyles={cardStyles}>
```

Comment:
Without this change, it was appearing like this in mobile views

<img width=300 src="https://user-images.githubusercontent.com/16595804/198016821-9ecec6d8-8804-405c-9325-a2b77b184b27.png" />

<img width="300" src="https://user-images.githubusercontent.com/16595804/198016827-f10ab3dd-c2d4-458d-ba6d-e6d83dd1ba49.png" />